src/content/basics/azure-architecture/index.md

```diff
@@ -0,0 +1,44 @@
++++
+title = "The Giant Swarm Azure Architecture"
+description = "Architecture Overview showing how Giant Swarm is set up on Microsoft Azure"
+date = "2019-07-16"
+weight = 20
+type = "page"
+categories = ["basics"]
++++
+
+# The Giant Swarm Azure Architecture
+
+Giant Swarm's Architecture is split into two logical parts, one being the Control Plane and the other being multiple Tenant Clusters. Among other services (for details see Service Architecture below), the control plane runs our [Azure Operator](https://github.com/giantswarm/azure-operator), which handles the full lifecycle management of Tenant Clusters.
+
+## Giant Swarm control plane
+
+![Control Plane Architecture](/img/architecture-azure-control-plane.png)
+
+The above diagram shows our control plane setup running with two bastion hosts in different availability zones (???). All instances are in a private subnet and only accessible through the bastion hosts. Bastions hosts are available via Azure VPN connection only.
+
+The cluster has several APIs and Interfaces. The Kubernetes API of the Control Plane is only available to Giant Swarm operations personel and only through the Bastion hosts. 
+
+The Giant Swarm API, Monitoring and Alerting frontends as well as our Web Management Interface, Happa, are exposed through the Control Plane Ingress Controller, which sits behind a public load balancer (???).
+
+## Giant Swarm tenant cluster
+
+![Tenant Cluster Architecture](/img/architecture-azure-tenant-cluster.png)
+
+Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our Happa interface, you can start Tenant Clusters of different sizes. There's a selection of recommended Azure instance types (???), which can be adjusted if needed. 
+
+Each cluster resides in its own VPC (WHAT IS IT IN AZURE?). All Azure machines are in their own private subnet ([Networking Deepdive](https://blog.giantswarm.io/deep-dive-into-kubernetes-networking-in-azure/)). There are two possible access routes into the cluster. 
```

Comment:
I think its Virtual Network but I'll check. Why can't they use the same naming as AWS!

Great to link to Roma's post. <3

Reply:
Virtual Network fine, just not a VPC. VPC as a term does not exist. I think. I am glad to be proven wrong :)